styles.css

```diff
@@ -183,14 +220,15 @@ li a:hover {
 */
 
 .vision-mission-values {
+  /* border: 1px red solid; */
```

Comment:
Once you are done with debugging border (1px solid red), remove it. don't comment -- delete.